### DISCUSSION TITLE
Suspicious program

### DISCUSSION THREAD

**Discussion by Tallestrom** [Ruben2776/PicView#158] [General]
I'm no programmer but hybrid-analysis.com Falcon Sandbox triggered on this big time.  For such a simple program it sure does a lot of suspicious things behind the scenes.

https://tinyurl.com/264pl2ll

**Comment by Ruben2776** [comment]
PicView is virus scanned when published through sources such as Chocolatey: 
https://www.virustotal.com/gui/file/8c9d92f7744e17760196970bbfa6db76ef762e0c8b5e676cc0054f94f003d601/detection/f-8c9d92f7744e17760196970bbfa6db76ef762e0c8b5e676cc0054f94f003d601-1722966492 and https://www.virustotal.com/gui/file/8bb3721dcc9b77833020ae8fc4d555e45611a95f0efc0346a7745f426da7a2c5/detection/f-8bb3721dcc9b77833020ae8fc4d555e45611a95f0efc0346a7745f426da7a2c5-1723383435
It's also published through [Uptodown](https://picview.en.uptodown.com/windows), who does [automatic virus scan](https://support.uptodown.com/hc/en-us/articles/360062090652-Is-Uptodown-safe)
